src/Microsoft.DotNet.XHarness.CLI/CommandArguments/Android/AndroidInstallCommandArguments.cs

```diff
@@ -34,6 +43,18 @@ protected override OptionSet GetTestCommandOptions() => new()
                 "device-id=", "Device where APK should be installed",
                 v => DeviceId = v
             },
+            {
+                "boot-timeout=", "Timeout in seconds to wait for a device after boot competion",
```

Comment:
Please check the timeouts in iOS commands, we use `TimeSpan` and we parse both integers for seconds or strings like `00:02:30`.